CONDUCT.md

```diff
@@ -0,0 +1,98 @@
+# Scikit-Optimize Code of Conduct
+
+Scikit-Optimize is an engaged and respectful community made up of people
+from all over the world. Your involvement helps us to further our
+mission and to create an open platform that serves a broad range of
+communities, from research and education, to journalism, industry and
+beyond.
+
+Naturally, this implies diversity of ideas and perspectives on often complex
+problems. Disagreement and healthy discussion of conflicting viewpoints is
+welcome: the best solutions to hard problems rarely come from a single angle.
+But disagreement is not an excuse for aggression: humans tend to take
+disagreement personally and easily drift into behavior that ultimately degrades
+a community. This is particularly acute with online communication across
+language and cultural gaps, where many cues of human behavior are unavailable.
+We are outlining here a set of principles and processes to support a
+healthy community in the face of these challenges.
+
+Fundamentally, we are committed to fostering a productive, harassment-free
```

Comment:
I am trying to understand what this paragraph means. Does it mean that it is not a set of hard rules? And also that one should use common sense for situations beyond the code of conduct?

Reply:
The way I see it is to encourage the reader not to read this like a legal text that tries to solve every possible event, instead use common sense for where there are gaps. It doesn't mean that application of this COC is going to depend on who you are or who you are friends with.

Reply:
> ... encourage the reader not to read this like a legal text that tries to solve every possible event, instead use common sense for where there are gaps. It doesn't mean that application of this COC is going to depend on who you are or who you are friends with

Tbh the way you put it in your comment appears to be more clear than what it is now in COC :) Consider adding it instead of current text.